app/chapters/[slug]/genesis/genesis-pt2/page.tsx

```diff
@@ -58,7 +59,14 @@ export default function Genesispt2() {
           <div
             className="genesis"
             dangerouslySetInnerHTML={{ __html: genesis.body.html }}
-          ></div>
+          >
+          </div>
+          <div className='genesis'>
+          <pre><code className='language-shell pl-2 pr-2 items-center flex justify-between'>
```

Comment:
```suggestion
          <pre><code className='language-shell pl-2 pr-0 items-center flex justify-between'>
```
Too much padding on the right.